monad-rpc/src/handlers/debug.rs

```diff
@@ -268,6 +294,31 @@ pub struct MonadDebugTraceTransactionParams {
     tracer: TracerObject,
 }
 
+#[derive(Serialize, Debug, schemars::JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct MonadCallFrameLog {
+    address: EthAddress,
+    topics: Vec<FixedData<32>>,
+    data: UnformattedData,
+    position: Quantity,
```

Comment:
`position` should be u64. Quantity serializes to a hex string, but the response should be an integer